.github/workflows/kata-deploy-test.yaml

```diff
@@ -48,18 +48,16 @@ jobs:
           - rootfs-initrd
           - shim-v2
     steps:
-      # As Github action event `issue_comment` does not provide the right ref
```

Comment:
I suggest to rephrase that comment and keep it as it helps a lot to understand why the heck it needs to get the ref.